spec/glossary.adoc

```diff
@@ -2,12 +2,23 @@
 = Glossary
 
 [glossary]
+[[active_rule]]Active Rule:: An active rule is an _instance_ of a <<rule,rule>>. An active rule may have different attributes than the rule. For example severity and parameters values can be overridden in quality profiles, so that different <<sonar_project,Sonar Projects>> can be analyzed for the same rule, but with different behavior/threshold. Active rules are part of the input for <<sensor,sensors>>.
 [[binding]]Binding:: In <<connected_mode>>, this is the association between a <<configuration_scope,configuration scope>> and a <<sonar_project,Sonar project>>, for a given <<connection,connection>>.
 [[binding_clue]]Binding Clue:: For a <<configuration_scope,configuration scope>>, this is any element that could help to guess what should be the <<binding,binding>> in <<connected_mode,connected mode>>. A configuration scope can have multiple binding clues, partial, or even contradictory. They will be processed during xref:binding_suggestion.adoc#binding_suggestion[binding suggestion computation].
 [[configuration_scope]]Configuration Scope:: Represent an abstract IDE element that can be configured, and possibly <<binding,bound>>. Usually a project (Eclipse, IntelliJ), module (IntelliJ), solution (VS, Rider), or workspace folder (VSCode).
 [[connected_mode]]Connected Mode:: The use of SonarLint associated with a SonarQube server or SonarCloud. See also <<standalone_mode,standalone mode>>.
 [[connection]]Connection:: A channel of communication between SonarLint and a particular SonarQube server or SonarCloud. See also <<connected_mode,connected mode>>.
 [[connection_config]]Connection Configuration:: All attributes that will be used to instantiate and operate a <<connection,connection>>. Usually provided by the IDE settings. It includes credentials, endpoint URL, timeouts, proxy...
+[[effective_rule_details]]Effective Rule Details:: <<rule_details>> taking into account the <<binding,binding>> and the <<finding_context,finding context>>. Some attributes can be different from the <<raw_rule_details, raw rule details>>.
+[[finding]]Finding:: Generic concept that includes all things reported by <<sonar_analyzer,Sonar analyzers>>: issues, security hotspots, taint vulnerabilities, ...
+[[finding_context]]Finding Context:: When reporting a finding, <<sonar_analyzer,Sonar analyzers>> can optionally provide a context key. This context key will be used to adapt the <<rule_details>> to give more specific remediation guidelines. For example, we can have framework specific context (spring vs struts).
```

Comment:
Unrelated to the current work, but reading "Finding Context", I don't know why I thought about secondary locations or flows. I think it's also part of the context, so maybe this concept could be enriched later